docs/CHANGES.md

```diff
@@ -13,6 +13,28 @@ Changelog
 
 <!-- TOWNCRIER -->
 
+3.20.0 (2022-06-22)
+===================
+
+
+Bugfixes
+--------
+
+- Fix gnutls error on the webserver `Peer's certificate does not allow digital signatures. Key usage violation detected.` by adding the key_usage digitalSignature to the CSR.
+  [#1182](https://github.com/pulp/pulp_installer/issues/1182)
+- Fix an issue where by the pulp_webserver role would fail on "Copy Snippets" if it were run after pulp_devel is run.
+  [#1186](https://github.com/pulp/pulp_installer/issues/1186)
+
+
+Misc
+----
+
+- [#1160](https://github.com/pulp/pulp_installer/issues/1160)
+
+
+----
```

Comment:
I believe you need to re-run the towncrier, more changelogs were added

Reply:
Thank you! I update my PR.